src/commands/explain.rs

```diff
@@ -0,0 +1,45 @@
+use clap::Parser;
+use wdl::ast::v1::lint as ast_lint;
+use wdl::grammar::v1::lint as grammar_lint;
+
+/// Arguments for the `explain` subcommand.
+#[derive(Parser, Debug)]
+#[command(author, version, about)]
+pub struct Args {
+    /// The rule name to explain.
+    #[arg(required = true)]
+    pub rule_name: String,
+}
+
+pub fn explain(args: Args) -> anyhow::Result<()> {
+    let name = args.rule_name;
+
+    let rules = grammar_lint::rules()
```

Comment:
It seems odd to me that we're iterating over both lists twice. Can this be streamlined?